data/jibauni/rebel jibauni missions.txt

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2024 by Loymdayddaud
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Rebel Jibauni: Qui-ni"
+	minor
+	name "Rebel Jibauni transport"
+	description "You bumped into a Rebel Jibauni in <origin>'s spaceport and agreed to take it to <destination>."
+	source
+		government "Jibauni (Rebel)"
+	destination Kanlil
+	to offer
+		random < 3
+	on offer
+		conversation
+			`Stepping off of the <ship> into the busy spaceport of <origin>, you see a Rebel Jibauni coming straight towards you with a purposeful look on its face. Looking around, you also see a patrol of Rebel Jibauni soldiers who seem to be searching the spaceport.`
+			choice
+				`	(Wait for the Jibauni to arrive.)`
+				`	(Leave.)`
+					decline
+			`	The Jibauni is coming faster now. When it gets within speaking distance you hear it ask, "Can anyone take me to <planet>? I need to leave immediately."`
+			choice
+				`	(Offer to transport the Jibauni.)`
+				`	(Ignore the Jibauni.)`
+					decline
+			`	You ask the Jibauni if you would fit the bill for transportation. It immediately thanks you and pulls you towards the <ship>, seemingly wanting to go.`
+			choice
+				`	(Follow the Jibauni and leave with it.)`
+					goto followandleave
+				`	"Whoa, not so fast there."`
+			`	"We've got to go now," the Jibauni exclaims. "Come on!" It pulls you into your ship and forces you to launch.`
+				launch
+			label followandleave
+			`	You take off upon reaching your ship. Rebel Jibauni ships launch with you.`
+				launch
+	on enter
+		"old reputation: Rebel Jibauni" = "reputation: Jibauni (Rebel)"
+		"reputation: Jibauni (Rebel)" <?= -1
+		conversation
+			`The Jibauni seems somewhat calmer now. "My name is Qui-ni," it says. "You are taking me to <destination>, right?"`
+			choice
+				`	"Well, I did leave with you."`
+				`	"Not willingly."`
+					goto notwilling
+			`	"Good. Now, please evade the Rebel Jibauni who are chasing us. Let's go immediately."`
+				goto end
+			label notwilling
+			`	"Really? Well, you can take me to <destination> immediately and be rid of me."`
+			label end
```

Comment:
Should there be an `on abort` message if the player is _super_ not willing and calls the Jibauni's bluff? Maybe like
```suggestion
			label end
		on abort
			dialog `The Jibauni looks at you in shock as you get rid of them earlier than expected.`			
```